react-app/src/App.js

```diff
@@ -126,13 +147,13 @@ const reverseGeocode = (lat, long) => {
     return `Landing location unknown, unknown coordinates`
   } else {
     const geoUrl = `https://api.geoapify.com/v1/geocode/reverse?lat=${lat}&lon=${long}&apiKey=978b6815c5e545ba9b0dc6b2cf38f294`
-    fetch(geoUrl, {
+    return fetch(geoUrl, {
```

Comment:
now this function returns a promise because it fetches data asynchronously.  So we can call `reverseGeocode().then()` - read more about promises here:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise